python_modules/dagster/dagster/cli/pipeline.py

```diff
@@ -151,13 +151,13 @@ def print_solid(printer, solid):
 
         printer.line('Outputs:')
 
-        for output_def in solid.output_defs:
+        for output_def in solid.definition.output_defs:
```

Comment:
Is this going to break? Not sure.

If it breaks, fix. If not, can you add a type check at the beginning on print_solid?